lib/adapters/stochastic.js

```diff
@@ -27,11 +29,11 @@ var Read = Juttle.proc.source.extend({
         }
         this.type = options.type;
         if (this.type && this.type !== 'metric' && this.type !== 'event') {
-            throw this.compile_error('RT-METRICS-OR-EVENTS');
+            throw new errors.compileError('RT-METRICS-OR-EVENTS');
         }
 
-        this.handleTimeOptions(options);
-        this.setDefaultTimeRange(true);
+        this.from = options.from || NOW;
+        this.to = options.to || NOW;
```

Comment:
Is there a use case for Stochastics that go from now to now? Seems like it'd be better to require `from`/`to`.